rewrite-maven/src/main/java/org/openrewrite/maven/cache/RocksdbMavenPomCache.java

```diff
@@ -164,7 +158,13 @@ public RocksdbMavenPomCache(@Nullable File workspace) {
         }
 
         byte[] key = serialize(repo.toString() + ":" + artifactCoordinates);
-        Optional<RawMaven> rawMavenEntry = deserializeRawMaven(cache.get(key));
+        Optional<RawMaven> rawMavenEntry = null;
+        try {
+            rawMavenEntry = deserializeRawMaven(cache.get(key));
+        } catch (Exception ex) {
+            // upon serialization exceptions delete the cache entry and force it to be re-downloaded
+            cache.database.delete(key);
+        }
```

Comment:
I think we can just change deserializeRawMan() such that it catches the serialization errors and just returns null (which is a cache miss) 

Then the logic will fall through, download the pom file, serialize it (in the new format), and then replace the one that is there.

Reply:
We can do something similar for the other deserialize methods.